hack/ocm-minikube.sh

```diff
@@ -141,7 +154,7 @@ exit_stack_push unset -f ocm_registration_operator_spoke
 ocm_registration_operator_deploy_hub()
 {
 	set -- $hub_cluster_name
-	kubectl --context $1 apply -f https://raw.githubusercontent.com/kubernetes/cluster-registry/master/cluster-registry-crd.yaml
+	#kubectl --context $1 apply -f https://raw.githubusercontent.com/kubernetes/cluster-registry/master/cluster-registry-crd.yaml
```

Comment:
(note) This was an older requirement, which we never saw used, so good to have it out. Long time back, when this was not added one of the OCM operators on the hub used to throw consistent errors logs (still worked, just logs). At some point we may need to ensure this is not the case any longer.

Reply:
I haven't seen an error log in the one or two runs I have done. I am pretty sure it is not required for minikube versions > 1.22 which have the k8s version with this CRD already installed.